web/src/containers/P2P/TradesHistory/index.tsx

```diff
@@ -120,8 +122,8 @@ const TradesHistory: FC = (): ReactElement => {
     return (
         <div>
             {fetching && <div className="text-center"><Spinner animation="border" variant="primary" /></div>}
-            {list.length ? renderContent() : null}
-            {!list.length && !fetching ? <p className="pg-history-elem__empty">{intl.formatMessage({id: 'page.noDataToShow'})}</p> : null}
+            {list.length ? renderContent : null}
```

Comment:
Create a separate method using useMemo for this block

Reply:
Fixed, thanks.